Source/Fields.H

```diff
@@ -45,11 +45,11 @@ namespace warpx::fields
         vector_potential_fp_nodal,
         vector_potential_grad_buf_e_stag,
         vector_potential_grad_buf_b_stag,
-        hybrid_electron_pressure_fp,
-        hybrid_rho_fp_temp,
-        hybrid_current_fp_temp,
-        hybrid_current_fp_ampere,
-        hybrid_current_fp_external,
+        hybrid_electron_pressure_fp, //!< Used with Ohm's law solver. Stores the electron pressure
+        hybrid_rho_fp_temp,          //!< Used with Ohm's law solver. Stores the time interpolated/extrapolated charge density
+        hybrid_current_fp_temp,      //!< Used with Ohm's law solver. Stores the time interpolated/extrapolated current density
+        hybrid_current_fp_plasma,    //!< Used with Ohm's law solver. Stores plasma current calculated as J_plasma = curl x B / mu0 - J_ext
+        hybrid_current_fp_external,  //!< Used with Ohm's law solver. Stores external current
```

Comment:
Need to use the inline format for the comment, because of the macro used:
```suggestion
        hybrid_electron_pressure_fp, /**< Used with Ohm's law solver. Stores the electron pressure */
        hybrid_rho_fp_temp,          /**< Used with Ohm's law solver. Stores the time interpolated/extrapolated charge density */
        hybrid_current_fp_temp,      /**< Used with Ohm's law solver. Stores the time interpolated/extrapolated current density */
        hybrid_current_fp_plasma,    /**< Used with Ohm's law solver. Stores plasma current calculated as J_plasma = curl x B / mu0 - J_ext */
        hybrid_current_fp_external,  /**< Used with Ohm's law solver. Stores external current */
```